CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
 # [Versions](https://mui.com/versions/)
 
+## v6.0.0-beta.0
+
+<!-- generated comparing v6.0.0-alpha.14..next -->
+
+_Jul 10, 2024_
+
+A big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎉 This is the first beta release of v6! A huge thanks to everyone who helped to make this happen! We're targeting the 1st of August for a stable release, and will use the next two months to polish v6, and ease the migration from v5. You can follow the [v6 stable milestone](https://github.com/mui/material-ui/milestone/42) for more details. We now encourage any new projects to start on v6.
```

Comment:
I copied this from the v5.0.0-beta.0 release. @DiegoAndai let me know if I should remove or update some info like the tentative date (it's completely made up).